apps/datahub/src/app/record/record-apis/record-apis.component.html

```diff
@@ -1,67 +1,99 @@
-<div class="container-lg px-4 lg:mx-auto">
-  <p
-    class="font-title text-[28px] font-medium mt-8 mb-5 text-title text-center sm:mt-12 sm:mb-[-22px] sm:text-left"
-    translate
+<ng-container *ngIf="apiLinks$ | async as apiLinks">
+  <div
+    class="flex flex-row gap-x-4 items-center justify-center md:justify-start"
   >
-    record.metadata.api
-  </p>
+    <p
+      class="font-title text-[28px] font-medium text-title text-center sm:text-left"
+      translate
+    >
+      record.metadata.api
+    </p>
+    <div class="flex flex-row gap-x-4 items-center" *ngIf="hasSteps">
+      <gn-ui-button
+        [style.--gn-ui-button-rounded]="'100%'"
+        [style.--gn-ui-button-width]="'0.5rem'"
+        [style.--gn-ui-button-height]="'0.5rem'"
+        [style.--gn-ui-button-padding]="'0.7rem'"
+        [type]="isFirstStep ? 'default' : 'outline'"
+        [disabled]="isFirstStep"
+        (buttonClick)="slideToPrevious()"
+      >
+        <mat-icon
+          class="material-symbols-outlined text-[0.9rem] text-center pt-[0.3rem]"
+        >
+          arrow_back
+        </mat-icon>
+      </gn-ui-button>
+      <gn-ui-button
+        [style.--gn-ui-button-rounded]="'100%'"
+        [style.--gn-ui-button-width]="'0.5rem'"
+        [style.--gn-ui-button-height]="'0.5rem'"
+        [style.--gn-ui-button-padding]="'0.7rem'"
+        [type]="isLastStep ? 'default' : 'outline'"
+        [disabled]="isLastStep"
+        (buttonClick)="slideToNext()"
+      >
+        <mat-icon
+          class="material-symbols-outlined text-[0.9rem] text-center pt-[0.3rem]"
+        >
+          arrow_forward
+        </mat-icon>
+      </gn-ui-button>
+    </div>
+  </div>
   <gn-ui-carousel
+    [itemsLength]="apiLinks.length"
+    (stepsChanged)="carouselStepsHasChanged($event)"
     containerClass="gap-4 py-10"
     stepsContainerClass="right-[var(--container-outside-width)] bottom-[calc(100%-12px)] top-auto"
   >
     <gn-ui-api-card
-      *ngFor="
-        let link of facade.apiLinks$ | async;
-        let first = first;
-        let last = last
-      "
+      *ngFor="let link of apiLinks"
       [title]="link.name"
       [link]="link"
       [currentLink]="selectedApiLink"
-      class="w-80"
+      class="w-full md:w-[20rem]"
       [ngClass]="{
-        'mr-[var(--container-outside-width)]': last,
-        'ml-[var(--container-outside-width)]': first,
```

Comment:
What was the reason for these layout changes? I think that because of them a scrollbar now appears at the bottom of the page when there is an overflow of the carousel

Reply:
I didn't see the scroll bar until now, I'll take a look at that.

Reply:
I fixed it just by setting overflow-hidden tailwind class on the record-page.

Reply:
but why keep that change? I mean, in order to refactor something there needs to be an expected gain out of it, and I'm not sure I see what that gain is (except the removal of two lines of code maybe?)

Also the `w-full` class shows cards as full width now, is that intentional?
![image](https://github.com/geonetwork/geonetwork-ui/assets/10629150/63b3ddba-1317-4133-aa92-b3e2684cbab2)